### ISSUE TITLE
can not build on nightly-2-10

### ISSUE THREAD

**Issue by alissa-tung** [leanprover-community/mathlib4#2189]
```
./lake-packages/mathlib4/././Mathlib/Tactic/Have.lean:46:18: error: unknown constant 'Lean.Elab.Tactic.elabTerm.go'
```

See https://github.com/leanprover/lean4/commit/3c562c1a9b401683a51a3854efd99c44b325592b

**Comment by jcommelin** [comment]
Fixed by #2188

**Comment by alissa-tung** [comment]
https://github.com/leanprover-community/mathlib4/pull/2188
